.pre-commit-hooks.yaml

```
@@ -8,6 +8,15 @@
   language: python
   pass_filenames: false
 
+- id: ggshield-iac
+  name: ggshield-iac (pre-commit)
+  entry: ggshield-iac
```

```suggestion
  entry: ggshield
```
`entry` is the executable to run, cf https://pre-commit.com/#new-hooks